packages/apidom-ns-workflows-1/src/refractor/visitors/workflows-1/step/ParametersVisitor.ts

```diff
@@ -0,0 +1,12 @@
+import stampit from 'stampit';
+
+import StepParametersElement from '../../../../elements/nces/StepParameters';
+import BaseParametersVisitor from '../ParametersVisitor';
+
+const ParametersVisitor = stampit(BaseParametersVisitor, {
```

Comment:
No need for doing this. There is only one Parameter Object list inside the spec. The BaseParameterVisitor, should be merged into this file: https://github.com/swagger-api/apidom/pull/3471/files#diff-2c8b689e039b2987e9c7a4e8ae33567fa4edcbb89f72cc2f5d614c57927cf873

Reply:
Changes applied - thanks